### DISCUSSION TITLE
Welcome to the sonoma-import discussion!

### DISCUSSION THREAD

**Discussion by zyphlar** [zyphlar/sonoma-import#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions
  * Share ideas.
  * Engage/plan
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you're interested in doing.

**Comment by trogdoor17** [comment]
Hello! This is a great project. I'm not sure how much I can contribute but I would like to be involved however I can. I don't know too much about sql and databases (Or Github tbh), but I have some experience with ArcGIS and I'm looking to learn some more!
